lib/dashboard.dart

```diff
@@ -131,3 +148,68 @@ class BackupSeedCard extends StatelessWidget {
         ));
   }
 }
+
+@immutable
+class PaymentListItem extends StatelessWidget {
```

Comment:
🤔 I suggest extracting that into a separate class.

Reply:
I suppose you mean a separate file :)
I moved it and aligned the naming a bit.